mlflow/utils/autologging_utils/safety.py

```diff
@@ -26,19 +26,6 @@
 _AUTOLOGGING_PATCHES = {}
 
 
-def try_mlflow_log(fn, *args, **kwargs):
-    """
-    Catch exceptions and log a warning to avoid autolog throwing.
-    """
-    try:
-        return fn(*args, **kwargs)
-    except Exception as e:
-        if is_testing():
-            raise
-        else:
-            warnings.warn("Logging to MLflow failed: " + str(e), stacklevel=2)
```

Comment:
Can we fix this lint error by removing `import warnings` in this file?

https://github.com/mlflow/mlflow/runs/4070952017?check_suite_focus=true#step:6:26

```
mlflow/utils/autologging_utils/safety.py (7,0): [W0611 unused-import] Unused import warnings
```